### PR TITLE
docs(README): refactor README.md

### PR DESCRIPTION
**Pull request by jia200x**
## Contribution description

This PR:
- Adapt line lengths of README to 80 chars.
- Adds a Getting Started section to the README. It shows the quick setup to install Open-SIR.
- Removes parts that are already included in the official documentation.

## Testing

- Check if the README looks OK and if all links are valid

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,10 @@
 [![Documentation Status][rtd]](https://open-sir.readthedocs.io/en/latest)
 
 
-Open-SIR is an Open Source Python project for modelling pandemics and infectious diseases using Compartmental Models, such as the widely used [Susceptible-Infected-Removed (SIR) model](http://rocs.hu-berlin.de/corona/docs/forecast/model/#classic-sir-dynamics). 
+Open-SIR is an Open Source Python project for modelling pandemics and
+infectious diseases using Compartmental Models, such as the widely used
+[Susceptible-Infected-Removed (SIR)
+model](http://rocs.hu-berlin.de/corona/docs/forecast/model/#classic-sir-dynamics).
```
Comment (sasalatart) [inline README.md:17]: Maybe you can add this link to the top of the file, like these lines:
https://github.com/open-sir/open-sir/pull/60/files#diff-04c6e90faac2675aa89e2176d2eec7d8R4

The markdown would look better imo

File: README.md [hunk 2]
```diff
@@ -20,189 +23,52 @@ The current stage of the software is *Alpha*.
 - Calculation of confidence intervals
 - CLI for interfacing with non Python environments (Bash, Node.JS, Matlab, etc).
 
-So far, Open-SIR provides an implementation of the SIR model and the novel [SIR-X model, developed by Maier and Dirk](https://science.sciencemag.org/content/early/2020/04/07/science.abb4557.full) from the [Robert Koch Institut](http://rocs.hu-berlin.de/corona/docs/forecast/model/#sir-x-dynamics-outbreaks-with-temporally-increasing-interventions).
+So far, Open-SIR provides an implementation of the SIR model and the novel
+[SIR-X model, developed by Maier and
+Dirk](https://science.sciencemag.org/content/early/2020/04/07/science.abb4557.full)
```
Comment (sasalatart) [inline README.md:28]: Same here about link aliases

File: README.md [hunk 3]
```diff
@@ -20,189 +23,52 @@ The current stage of the software is *Alpha*.
...
 
-* Python >= 3.2
-* Numpy
-* Sklearn
-* Scipy
-
-## Installation
-Open-SIR and all its dependencies can be installed from the repository using
-`pip`:
+Welcome to Open-Sir! Install Open-SIR using `pip`
```
Comment (sasalatart) [inline README.md:37]:
```suggestion
Welcome to Open-Sir! Install Open-SIR using `pip`:
```

**Comment by felipehuerta17** [comment]
What do you think on adding something like this: 

image

I have to update the nb so the axis have proper labels, but I think that it summarizes the whole project very well!

**Comment by jia200x** [comment]
> What do you think on adding something like this:

Sure! Let's do it.

**Comment by jia200x** [comment]
@sasalatart all other comments were addressed

**Comment by felipehuerta17** [comment]
@jia200x Alright, working on it now!

**Comment by felipehuerta17** [comment]
Here it is!
sir_x_guangdong

**Comment by felipehuerta17** [comment]
Sorry the previous one has too few quality 
image

Please use the following one with 300 dpi elsevier journal artwork specification

sir_x_guangdong

**Comment by felipehuerta17** [comment]
@jia200x yet another update, sorry man! the previous one has misleading axis titles and title
sir_x_guangdong

**Comment by jia200x** [comment]
done!

**Comment by felipehuerta17** [comment]
To generate a link & improve visualization
sir_x_guangdong

**Comment by felipehuerta17** [comment]
sir_x_guangdong

**Comment by jia200x** [comment]
done!

**Pull request merged**